docs/databases/hadoop/how-to-install-and-set-up-hadoop-cluster/index.md

```diff
@@ -349,13 +365,13 @@ For 2GB nodes, a working configuration may be:
 
 3.  Unzip the binaries, rename the directory, and exit **node1** to get back on the node-master:
 
-        tar -xzf hadoop-2.8.1.tar.gz
-        mv hadoop-2.8.1 hadoop
+        tar -xzf hadoop-3.1.2.tar.gz
+        mv hadoop-3.1.2 hadoop
         exit
 
 4.  Repeat steps 2 and 3 for **node2**.
 
-5.  Copy the Hadoop configuration files to the slave nodes:
+5.  Copy the Hadoop configuration files to the worker nodes:
```

Comment:
both terms are used interchangeably and are technically correct, if given the choice I lean in favor of a softer approach